packages/experimental/src/components/UserProfileImage/UserProfileImage.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright IBM Corp. 2020, 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Canary } from '../_Canary';
+import { pkg } from '../../settings';
+const componentName = 'UserProfileImage';
+
+import {
+  User16,
+  User20,
+  User24,
+  User32,
+  Group16,
+  Group20,
+  Group24,
+  Group32,
+} from '@carbon/icons-react';
+
+//import { expPrefix } from '../../global/js/settings';
+import cx from 'classnames';
+
+const blockClass = `${pkg.prefix}-user-profile-avatar`;
+
+export const UserProfileImage = !pkg.isComponentEnabled(componentName)
+  ? // Return canary if not released or flag not set
+    () => <Canary component={componentName} />
+  :({
+  backgroundColor,
+  icon = 'user',
+  initials,
+  image,
+  size = 'x-large',
+  theme
+}) => {
+
+  const icons = {
+    user: {
+      'x-small': <User16 />,
+      small: <User16 />,
+      medium: <User20 />,
+      large: <User24 />,
+      'x-large': <User32 />,
+    },
+    group: {
+      'x-small': <Group16 />,
+      small: <Group16 />,
+      medium: <Group20 />,
+      large: <Group24 />,
+      'x-large': <Group32 />,
```

Comment:
This is slightly wasteful, as it instantiates ten render functions for imported components that may never be used. What we could do instead is make this object a lookup from type/size to the imported component, then assign the component directly to a variable with a capital initial and use it in the JSX directly. See the corresponding suggestions below.

```suggestion
      'x-small': User16,
      small: User16,
      medium: User20,
      large: User24,
      'x-large': User32,
    },
    group: {
      'x-small': Group16,
      small: Group16,
      medium: Group20,
      large: Group24,
      'x-large': Group32,
```